spec/containers/Dockerfile.amazon_linux

```diff
@@ -0,0 +1,9 @@
+FROM amazonlinux:1
```

Comment:
Not sure what to do about this, but just looking at the repository I'd find the files here confusing. Maybe some README in `spec/containers` about what you're trying to do would help? Or a specific subdirectory like `spec/containers/wait_for_rc_environments/...` ?

Reply:
The README kinda hints what they're there for, but I'll make it more explicit.